rbac_api_synced.go

```diff
@@ -99,6 +99,14 @@ func (e *SyncedEnforcer) AddPermissionForUser(user string, permission ...string)
 	return e.Enforcer.AddPermissionForUser(user, permission...)
 }
 
+// AddPermissionsForUser adds multiple permissions for a user or role.
+// Returns false if the user or role already has one of the permissions (aka not affected).
+func (e *SyncedEnforcer) AddPermissionsForUser(user string, permission ...[]string) (bool, error) {
```

Comment:
What's the difference compared to the other `AddPermissionForUser(user string, permission ...string)`?